p/scripts/main.js

```diff
@@ -1606,3 +1612,111 @@ if (document.readyState && document.readyState !== 'loading') {
 	document.addEventListener('DOMContentLoaded', init_afterDOM, false);
 }
 // @license-end
+
+
+
+
+//Adds CSS class to an HTML element.
```

Comment:
Please move your code higher up, before the init stuff, and before the license-end tag

Reply:
Done.